services/API-service/src/api/event/event.service.ts

```diff
@@ -1075,8 +1070,6 @@ export class EventService {
         eventArea.mainExposureValue = +activeAlertArea.mainExposureValue;
         eventArea.firstIssuedDate = lastUploadDate.timestamp;
         eventArea.endDate = lastUploadDate.timestamp;
-        eventArea.stopped = false;
-        eventArea.manualStoppedDate = null;
```

Comment:
I guess userTrigger and date are simply filled with default values?

Reply:
@jannisvisser yes